cluster/gce/config-default.sh

```diff
@@ -114,9 +114,33 @@ KUBE_UP_AUTOMATIC_CLEANUP=${KUBE_UP_AUTOMATIC_CLEANUP:-false}
 
 # OpenContrail networking plugin specific settings
 NETWORK_PROVIDER="${NETWORK_PROVIDER:-none}" # opencontrail, flannel
-OPENCONTRAIL_TAG="${OPENCONTRAIL_TAG:-R2.20}"
-OPENCONTRAIL_KUBERNETES_TAG="${OPENCONTRAIL_KUBERNETES_TAG:-master}"
-OPENCONTRAIL_PUBLIC_SUBNET="${OPENCONTRAIL_PUBLIC_SUBNET:-10.1.0.0/16}"
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+
+if [[ $NETWORK_PROVIDER == "opencontrail" ]]; then
+    img=$(gcloud compute images list | grep debian-7-backports | awk '{print $1}')
```

Comment:
This feels like the tail wagging the dog.  Why does a network provider redefine all of these previous params?

These files are already way to complicated - this is guaranteed going to break.  Users will set flags up above and have them overridden here.   We (google) are not e2e testing this - what keeps it working?